### ISSUE TITLE
Sentry unable to scrub sensitive infomation in in case of raw body

### ISSUE THREAD

**Issue by rishabhsairawat** [getsentry/raven-ruby#948]
I have mentioned the password as a sensitive field in the configuration. But In case of invalid JSON, the sentry is sending the password as plain-text to server.

```
ActionDispatch::Http::Parameters::ParseError: 822: unexpected token at '{
"username":"abc.pqr",
"role": "admin"
"password": "Abc@123"
}'
```

**Comment by thiagotonon** [comment]
Any news on this issue? I'm having the same problem

**Comment by st0012** [comment]
thanks for reporting this issue, I've opened https://github.com/getsentry/raven-ruby/pull/984 for solving it. would you mind using

```ruby
gem 'sentry-raven', github: "getsentry/raven-ruby", branch: "fix-#948"
```

to give it a try?
